tests/core/activity/registry_test.py

```diff
@@ -62,3 +73,16 @@ async def test_on_message_sent(activity_registry, message_sent, command):
 
     command.before_matcher.assert_called_once()
     command.matches.assert_called_once()
+
+
+@pytest.mark.asyncio
+async def test_on_symphony_elements_action(activity_registry, elements_action, form):
```

Comment:
maybe we could add tests to cover different cases like the event not being an ElementsAction
and the matcher not returning true
(right now we have 100% lines coverage but not 100% branch coverage)

Reply:
will do